File: Array/27. Remove Element.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function (nums, val) {
    var indexNum = 0
    for (var i = 0; i < nums.length; i++) {
        if (nums[i] != val) {
            nums[indexNum] = nums[i];
            indexNum = indexNum + 1;
        }
    }

    for (var i = indexNum; i < nums.length; i++) {
        nums[i] = '_';
    }

    return indexNum
};

File: Practice/27. Remove Element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        left_index = 0

        for right_index in range(len(nums)):
            if nums[right_index] != val:
                nums[left_index] = nums[right_index]
                left_index = left_index + 1
            
        return left_index